__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const app = require('../app.js');
+const request = require('supertest');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const testData = require('../db/data/test-data/index.js');
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+describe('/api/topics', () => {
+  test('GET 200 - should respond with an array of topic objects, each with properties of "slug" and "description" ', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toEqual(expect.any(Array));
+        expect(body.topics);
```

Comment:
We need to check array length is greater than zero, otherwise if it is less than one we will auto pass as we won't get into the forEach()